src/components/common/BaseDropdownButton.vue

```diff
@@ -23,6 +65,48 @@ export default{
         onChange: {
             type: Function,
         },
+        multiselect: {
+            type: Boolean,
+        },
+        selected: {
```

Comment:
selected - sounds like boolean type, rename please to make it more understandable